src/System.Net.Http/src/System/Net/Http/Headers/HttpHeaderType.cs

```diff
@@ -0,0 +1,19 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.Net.Http.Headers
+{
+    [Flags]
+    internal enum HttpHeaderType
```

Comment:
Consider making this `: byte`.  Since it's embedded in classes frequently allocated (e.g. https://github.com/dotnet/corefx/pull/23091/files#diff-3b3d828e1c3f9a9d2e926e471d0ca21cR38), that'll help reduce their size.